.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug report
+description: Create a report to help us improve
+body:
+- type: checkboxes
+  attributes:
+    label: Prerequisites
+    options:
+    - label: I have written a descriptive issue title
+      required: true
+    - label: I have searched [issues](https://github.com/cake-build/cake/issues) to ensure it has not already been reported
+      required: true
+- type: dropdown
+  attributes:
+    label: Cake runner
+    options:
+      - Cake .NET Tool
+      - Cake Frosting
+      - Cake runner for .NET Framework
+      - Cake runner for .NET Core
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Cake version
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Operating system
+    options:
+      - Linux
+      - Windows
+      - MacOS
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Operating system architecture
+    options:
+      - 32-Bit
+      - 64-Bit
+  validations:
```

Comment:
- Should we have an `N/A` option? There are cases where the OS architecture is not relevant.

Reply:
Added `N/A` option